kafka-ballerina/build.gradle

```diff
@@ -251,20 +239,20 @@ task ballerinaBuild {
                 }
             }
         }
-//        // Doc creation and packing
-//        exec {
-//            workingDir project.projectDir
-//            environment "JAVA_OPTS", "-DBALLERINA_DEV_COMPILE_BALLERINA_ORG=true"
-//            if (Os.isFamily(Os.FAMILY_WINDOWS)) {
-//                commandLine 'cmd', '/c', "$distributionBinPath/ballerina.bat doc ${moduleName} && exit %%ERRORLEVEL%%"
-//            } else {
-//                commandLine 'sh', '-c', "$distributionBinPath/ballerina doc ${moduleName}"
-//            }
-//        }
-//        copy {
-//            from file("$project.projectDir/target/apidocs/${moduleName}")
-//            into file("$project.projectDir/build/docs_parent/docs/${moduleName}")
-//        }
+        // Doc creation and packing
+        exec {
+            workingDir project.projectDir
+            environment "JAVA_OPTS", "-DBALLERINA_DEV_COMPILE_BALLERINA_ORG=true"
+            if (Os.isFamily(Os.FAMILY_WINDOWS)) {
+                commandLine 'cmd', '/c', "$distributionBinPath/ballerina.bat doc ${moduleName} && exit %%ERRORLEVEL%%"
+            } else {
+                commandLine 'sh', '-c', "$distributionBinPath/ballerina doc ${moduleName}"
+            }
+        }
+        copy {
+            from file("$project.projectDir/target/apidocs/${moduleName}")
+            into file("$project.projectDir/build/docs_parent/docs/${moduleName}")
```

Comment:
PackageName isn't it?

Reply:
Fixed in ade2f48d896df48f11b9ccad56ab3197d502bb16